siliconcompiler/core.py

```diff
@@ -426,7 +426,11 @@ def create_cmdline(self, progname, description=None, switchlist=None, input_map=
             if 'fpga_partname' in cmdargs.keys():
                 self.set('fpga', 'partname', cmdargs['fpga_partname'], clobber=True)
             # running target command
-            self.load_target(cmdargs['option_target'])
+            try:
+                target_module = importlib.import_module(f'targets.{cmdargs["option_target"]}')
+                self.use(target_module)
+            except ModuleNotFoundError:
+                self.error(f'Target module {cmdargs["option_target"]} not found in $SCPATH or siliconcompiler/targets/.')
```

Comment:
This error message doesn't seem correct any more. Should probably be python_path and SCROOT